src/main/webapp/resources/js/pages/admin/components/announcements/AnnouncementsTable.jsx

```diff
@@ -23,35 +25,43 @@ export const AnnouncementsTable = forwardRef((props, ref) => {
 
   const columns = [
     {
-      title: i18n("iridaThing.id"),
-      width: 80,
-      dataIndex: "id",
+      title: il8n("announcement.control.priority"),
+      align: "center",
+      fixed: "left",
+      dataIndex: "priority",
+      render(item) {
+        return <IconFlag style={ item ? { color: blue6 } : { color: grey2 }} />;
```

Comment:
Can you update this line to read:
```javascript
render(hasPriority) {
  return <IconFlag style={{ color: hasPriority ? blue6 : grey2 }} />;
},
```
Just a little easier to quickly see what is going on.

Reply:
Please see c477ab71a043aa431205422872399e7968565e40.